deploy/addons/olm/olm.yaml.tmpl

```diff
@@ -326,8 +328,16 @@ spec:
                   requests:
                     cpu: 10m
                     memory: 50Mi
+                securityContext:
+                  runAsUser: 1000
```

Comment:
is this required ? this seems to give Root Access to the addon,
do u mind clarifying why we need to run with more special access ?

Reply:
I don't work on OLM directly I'm just a user and I did not write this definition. This comes from the official OLM release artifacts. 

But If I'm not mistaken root has uid `0`. This is not giving any special access to the container, it is doing quite the opposite. It ensures that the container can't run anything as root. 
`1000` is the first non-root user.  Even if the image is built in a way that it runs as root inside the container, this will force it to be non-root.

Reply:
that makes sense ! thanks for the clarifying